app/src/main/java/org/fossasia/susi/ai/chat/ChatPresenter.kt

```diff
@@ -41,6 +41,7 @@ class ChatPresenter(chatActivity: ChatActivity) : IChatPresenter, IChatModel.OnR
     private var micCheck = false
     var latitude: Double = 0.0
     var longitude: Double = 0.0
+    var deviceType: String = "Android"
```

Comment:
Why not private val?

Reply:
Also, type inference will get the type automatically, no need to specifically name it

Reply:
I've followed the existing code pattern, as used by the latitude and longitude since they relate to the same thing.

Reply:
No problem, we should leave the code better than we found it, make it private val